dbt/adapters/spark_cde/connections.py

```diff
@@ -84,6 +99,7 @@ class SparkCredentials(Credentials):
     server_side_parameters: Dict[str, Any] = field(default_factory=dict)
     retry_all: bool = False
     password: Optional[str] = None
+    auth_endpoint: Optional[str] = None
```

Comment:
Is this optional

Reply:
Not for CDE, but this also has livy code